src/main/java/io/github/lottetreg/httpserver/Server.java

```diff
@@ -2,9 +2,11 @@
 
 public class Server {
   public void start(ServerSocket serverSocket) {
-    Connection connection = serverSocket.acceptConnection();
-
-    new Reader().read(connection);
-    new Writer().write(connection, "HTTP/1.0 200 OK");
+    Connection connection;
+    while((connection = serverSocket.acceptConnection()) != null) {
+      HTTPRequest request = new Reader().read(connection);
+      new Writer().write(connection, "HTTP/1.0 200 OK\r\n\r\n" + request.body);
```

Comment:
This is a great place to refactor towards a `Response` abstraction, let's start here!